game-backend/src/MainController.ts

```diff
@@ -41,6 +44,32 @@ export default class MainController {
     }
   }
 
+  /**
+   * Looks up the connection id in the store.
+   * It returns the found connectionId or generates a new one respectively.
+   * @param requestedConnectionId The connection id the client requests
+   */
+  private getConnectionId(requestedConnectionId: string): ConnectionId {
+    if (
+      requestedConnectionId &&
+      this.store.userContexts.userContextIds.indexOf(requestedConnectionId)
```

Comment:
This probably won't work as expected in case the `requestedConnectionId` is at `userContextIds[0]`

Reply:
Thanks!